tests/tests/drush.yaml

```diff
@@ -12,31 +12,9 @@
 - ansible.builtin.import_playbook: drupal/drush-la.yaml
   vars:
     testname: "DRUSH integration {{ cluster_type|upper }}"
-    drupal_version: 9
-    db: mariadb-single
-    php_version: 8.0
     git_repo_name: drush.git
     project: ci-drush-la-{{ cluster_type }}
 
 - ansible.builtin.import_playbook: api/delete-project.yaml
   vars:
     project: ci-drush-la-{{ cluster_type }}
-
```

Comment:
Is this just removing the old drush site aliases because they don't work in newer versions of drupal we have in examples?

Reply:
:see_no_evil: reading the PR description answers this :smiling_face_with_tear: